src/forms/password/Password.tsx

```diff
@@ -130,6 +131,7 @@ export class Password extends React.PureComponent<PasswordProps, PasswordState>
                                     required={required}
                                     onKeyPress={onKeyPress}
                                     title={title}
+                                    onBlur={onBlur}
```

Comment:
Password component should already have a stoybook.  Add a Test case this.

onKeyPress -> triggers the change immediately after user types anything.  Why is onBlur needed here ? 
One problem with onBlur -> 

User types password, but never press tab -> Directly clicks Login buttin (in case of login screen) -> The onBlur wont fire.  Then what exact usecase we are solving here ? 

 if you set the Callback handler for onKeyPress, you keep password in your state and when user change it, you always have updated copy.  Please check if we really need onBlur ?

Reply:
The `onBlur` event is needed when we are using a `Password` component in the form.
Let's say the user clicked on the Password component but didn't provide any value for it, once the user leaves the component we can show an error as "Password cannot be blank" based on the `touched` property.

```
<Password
             onChange={props.handleChange}
             onBlur={props.handleBlur}
             name="password"
           />
```
Ref: https://formik.org/docs/api/formik

The test case is already available for this in the storybook.
https://github.com/EMCECS/clarity-react/blob/master/src/forms/password/Password.stories.tsx#L53